modules/admin-ui/src/main/webapp/scripts/shared/partials/wizards/new-event.html

```diff
@@ -473,7 +473,8 @@
                         data-disable-search-threshold="0"
                         data-search_contains="true"
                         ng-model="wizard.step.ud.workflow"
-                        ng-options="w.title for (obj, w) in wizard.step.workflows"
+                        ng-options="w.title for (obj, w) in wizard.step.workflows |
+                                  filter: wizard.getStateControllerByName('source').ud.type==='UPLOAD' ? 'upload-ng' : 'schedule-ng'"
```

Comment:
`-ng` was dropped in 5.x
```suggestion
                                  filter: wizard.getStateControllerByName('source').ud.type==='UPLOAD' ? 'upload' : 'schedule'"
```

Reply:
@mtneug, will you take the review of this pull request?

Reply:
@lkiesow yes.